lib/advert/ui/components/advert_card.dart

```diff
@@ -72,28 +72,38 @@ class AdvertCard extends HookConsumerWidget {
                       Expanded(
                         child: Column(
                           children: [
-                            Text(advert.title,
-                                textAlign: TextAlign.center,
-                                style: const TextStyle(
-                                  fontSize: 20,
-                                  fontWeight: FontWeight.bold,
-                                )),
+                            SingleChildScrollView(
```

Comment:
Maybe the usage of an AutoSizeText with multiline is more appropriate for a title